.github/workflows/nightly-test.yml

```diff
@@ -43,19 +45,23 @@ jobs:
         env:
           SANITIZER_ARG: ${{ matrix.sanitizer || 'OFF' }}
           EXPERIMENTAL: ${{ matrix.experimental || 'OFF' }}
+        working-directory: ${{ matrix.working_directory || github.workspace }}
         run: |
-          cmake -B build -DTILEDB_WERROR=ON -DTILEDB_SERIALIZATION=ON -DTILEDB_EXPERIMENTAL_FEATURES=$EXPERIMENTAL -DCMAKE_BUILD_TYPE=${{ matrix.config || 'Release' }} -DSANITIZER=$SANITIZER_ARG
+          cmake -B build -DTILEDB_VCPKG=ON -DTILEDB_WERROR=ON -DTILEDB_SERIALIZATION=ON -DTILEDB_EXPERIMENTAL_FEATURES=$EXPERIMENTAL -DCMAKE_BUILD_TYPE=${{ matrix.config || 'Release' }} -DSANITIZER=$SANITIZER_ARG
```

Comment:
#4348 will disable vcpkg for the nightly builds. This change will leave the old dependency management system untested.

Reply:
Let's follow up in sc-34078 and explicitly add matrix jobs to test what we want to cover.